experimental/DiscreteLog/DiscreteLog.jl

```diff
@@ -0,0 +1,24 @@
+module DiscreteLog
+
+  using GAP
```

Comment:
Why GAP?

Anyway, I don't think we need to put this into experimental, it can go straight into the main part. Perhaps into `src/NumberTheory/NmbThy.jl` ?